### PR TITLE
test-case: manual: check latency at the beginning

### PR DESCRIPTION
**Pull request by ClarexZhou**
Signed-off-by: clarex <clarex.zhou@intel.com>

### CODE REVIEW

File: test-case/manual/check-playback-capture-latency-at-the-beginning.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+# Test Description
+* Check playback latency at the beginning
+* Check capture latency at the begining
+
+## Test Case:
+1. `aplay -l` and `arecord -l` to check supported PCM
+2. Wait for about 3s till runtime PM is suspended, can check via `cat "/sys/bus/pci/devices/0000:00:1f.3/power/runtime_status"`, the pci device id can be find from `lspci` for Multimedia audio controller
+3. `aplay -Dhw:0,0 -r 48000 -f s16_le -c2 test.wav -vv -i` to trigger aplay from runtime PM suspended status, supported params can be dumped via `aplay -Dhw:0,0 --dump-hw-params`
+4. Check audiou output without obvious delay, latency at the begining is within 1s
```
Comment (fredoh9) [inline test-case/manual/check-playback-capture-latency-at-the-beginning.md:15]: In this manual test, it is hard to tell 1s+ or not. Arecord is even harder.
typo audiou->audio.

**Comment by marc-hb** [comment]
Can you please use a spell checker? There are many options. Most editors have plugins or something alike. I don't know which editor you use so here are two command line alternatives:
```
$ apt install spell
$ spell -n test-case/manual/check-playback-capture-latency-at-the-beginning.md 

2: piplines

9: begining

15: audiou
15: begining

17: piplines

18: Repeate
18: piplines

18: begining
```

```
../sof/scripts/checkpatch.pl -g HEAD
WARNING: Missing commit description - Add an appropriate one

ERROR: do not set execute permissions for source files
#13: FILE: test-case/manual/check-playback-capture-latency-at-the-beginning.md

WARNING: 'begining' may be misspelled - perhaps 'beginning'?
#26: FILE: test-case/manual/check-playback-capture-latency-at-the-beginning.md:9:
+* Check capture latency at the begining

WARNING: 'begining' may be misspelled - perhaps 'beginning'?
#32: FILE: test-case/manual/check-playback-capture-latency-at-the-beginning.md:15:
+4. Check audiou output without obvious delay, latency at the begining is within 1s

WARNING: 'begining' may be misspelled - perhaps 'beginning'?
#35: FILE: test-case/manual/check-playback-capture-latency-at-the-beginning.md:18:
+7. Repeate step 2 to step 5 for capture piplines with `arecord -Dhw:0,0 -r 48000 -f s16_le -c2 1.wav -vv -i`, then play this recorded 1.wav file to check latency at the begining is within 1s

total: 1 errors, 4 warnings, 23 lines checked

```

File: test-case/manual/check-playback-capture-latency-at-the-beginning.md (hunk 1 above)
Comment (ClarexZhou) [reply test-case/manual/check-playback-capture-latency-at-the-beginning.md:15]: Still can observe the delay with “-vv”  param for process check

**Comment by ClarexZhou** [comment]
@marc-hb  Thanks for check  and spell is verify useful. PR is updated now

File: test-case/manual/check-playback-capture-latency-at-the-beginning.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Latency test
+Check latency at the beginning is manual test for piplines which isn't setup loopback for alsabat test
+
+# Preconditions
+SOF is loaded successfully
+
+# Test Description
+* Check playback latency at the beginning
+* Check capture latency at the begining
```
Comment (fredoh9) [inline test-case/manual/check-playback-capture-latency-at-the-beginning.md:9]:
```suggestion
* Check capture latency at the beginning
```

File: test-case/manual/check-playback-capture-latency-at-the-beginning.md (hunk 1 above)
Comment (fredoh9) [inline test-case/manual/check-playback-capture-latency-at-the-beginning.md:15]:
```suggestion
4. Check audiou output without obvious delay, latency at the beginning is within 1s
```

File: test-case/manual/check-playback-capture-latency-at-the-beginning.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## Test Case:
+1. `aplay -l` and `arecord -l` to check supported PCM
+2. Wait for about 3s till runtime PM is suspended, can check via `cat "/sys/bus/pci/devices/0000:00:1f.3/power/runtime_status"`, the pci device id can be find from `lspci` for Multimedia audio controller
+3. `aplay -Dhw:0,0 -r 48000 -f s16_le -c2 test.wav -vv -i` to trigger aplay from runtime PM suspended status, supported params can be dumped via `aplay -Dhw:0,0 --dump-hw-params`
+4. Check audiou output without obvious delay, latency at the begining is within 1s
+5. Press ctrl and c button to stop aplay
+6. Repeat step 2 to step 5 for other playback piplines, such headset/HDMI/DP
+7. Repeate step 2 to step 5 for capture piplines with `arecord -Dhw:0,0 -r 48000 -f s16_le -c2 1.wav -vv -i`, then play this recorded 1.wav file to check latency at the begining is within 1s
```
Comment (fredoh9) [inline test-case/manual/check-playback-capture-latency-at-the-beginning.md:18]:
```suggestion
7. Repeate step 2 to step 5 for capture piplines with `arecord -Dhw:0,0 -r 48000 -f s16_le -c2 1.wav -vv -i`, then play this recorded 1.wav file to check latency at the beginning is within 1s
```

File: test-case/manual/check-playback-capture-latency-at-the-beginning.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+* Check playback latency at the beginning
+* Check capture latency at the begining
+
+## Test Case:
+1. `aplay -l` and `arecord -l` to check supported PCM
+2. Wait for about 3s till runtime PM is suspended, can check via `cat "/sys/bus/pci/devices/0000:00:1f.3/power/runtime_status"`, the pci device id can be find from `lspci` for Multimedia audio controller
+3. `aplay -Dhw:0,0 -r 48000 -f s16_le -c2 test.wav -vv -i` to trigger aplay from runtime PM suspended status, supported params can be dumped via `aplay -Dhw:0,0 --dump-hw-params`
+4. Check audiou output without obvious delay, latency at the begining is within 1s
+5. Press ctrl and c button to stop aplay
+6. Repeat step 2 to step 5 for other playback piplines, such headset/HDMI/DP
```
Comment (fredoh9) [inline test-case/manual/check-playback-capture-latency-at-the-beginning.md:17]:
```suggestion
6. Repeat step 2 to step 5 for other playback pipelines, such headset/HDMI/DP
```

File: test-case/manual/check-playback-capture-latency-at-the-beginning.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Latency test
+Check latency at the beginning is manual test for piplines which isn't setup loopback for alsabat test
```
Comment (fredoh9) [inline test-case/manual/check-playback-capture-latency-at-the-beginning.md:2]:
```suggestion
Check latency at the beginning is manual test for pipelines which isn't setup loopback for alsabat test
```

**Pull request merged**